xml/deploy_intro.xml

```diff
@@ -14,22 +14,11 @@
   </dm:docmanager>
  </info>
  <para>
-  Since &productname; 7, &ceph; services are deployed as containers instead of
-  packages. There are two ways to deploy the cluster:
- </para>
- <itemizedlist>
-  <listitem>
-   <para>
-    By using &cephadm;. Find more details in <xref linkend="deploy-cephadm"/>.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    On top of a &caasp; cluster. Find more details in
-    <xref linkend="deploy-rook"/>.
-   </para>
-  </listitem>
- </itemizedlist>
+  Since &productname; 7, &ceph; services are deployed as containers
+  instead of packages. This is deployed using &cephadm;. Find more details in
```

Comment:
```suggestion
  Since &productname; 7, &ceph; services are deployed as containers by using &cephadm;
  instead of RPM packages. Find more details in
```